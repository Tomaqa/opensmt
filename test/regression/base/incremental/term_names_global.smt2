(set-option :global-declarations true)
(set-logic QF_LIA)
(declare-fun o () Int)
(declare-fun i () Bool)
(push 1)
(assert (! (< o 0) :named c0))
(pop 1)
(assert (! i :named c0))
(check-sat)
