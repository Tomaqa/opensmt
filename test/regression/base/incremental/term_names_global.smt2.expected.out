(error "name c0 already exists")

(error "assertion returns an unknown sort")

sat
